features/tokendetails/impl/src/main/kotlin/com/tangem/feature/tokendetails/presentation/tokendetails/state/TokenDetailsBalanceBlockState.kt

```diff
@@ -0,0 +1,21 @@
+package com.tangem.feature.tokendetails.presentation.tokendetails.state
+
+import com.tangem.core.ui.components.buttons.actions.ActionButtonConfig
+import kotlinx.collections.immutable.ImmutableList
+
+sealed class TokenDetailsBalanceBlockState(open val actionButtons: ImmutableList<ActionButtonConfig>) {
```

Comment:
Можно же просто как abstract val определить, зачем дублировать в конструкторе?
```suggestion
sealed class TokenDetailsBalanceBlockState {

    abstract val actionButtons: ImmutableList<ActionButtonConfig>
```

Reply:
Сделал